Doc/library/socketserver.rst

```diff
@@ -129,6 +132,9 @@ server classes.
       :meth:`socketserver.ThreadingMixIn.server_close` now waits until all
       child processes and non-daemonic threads complete.
 
+      Add a new :attr:`socketserver.ForkingMixIn.block_on_close` class
+      attribute to opt-in for the old behaviour.
+
```

Comment:
It might read better it the addition were not a separate paragraph.   How about removing the blank line and rewording a bit, perhaps:

If necessary, set the new :attr:`socketserver.ForkingMixIn.block_on_close`  class
attribute to False to obtain the pre-3.7 behavior.

Reply:
Your proposed sentence doesn't explicitly say that the attribute is new in 3.7. I removed the empty line.